backend/templates/inputs/model-na-31-2.json

```diff
@@ -640,6 +640,6 @@
       "locality": "Heemdamseburg"
     }
   ],
-  "hash": "ED36 60EB 017A 0D3A D3EF 72B1 6865 F991 A36A 9F92 72D9 1516 39CD 422B 4756 D161",
-  "creation_date_time": "04-12-2024 12:08:55"
+  "hash": "ed36 60eb 017a 0d3a d3ef 72b1 6865 f991 a36a 9f92 72d9 1516 39cd 422b 4756 d161",
+  "creation_date_time": "04-12-2024 12:08"
```

Comment:
Why the change to lowercase and no seconds in the timestamp?

Reply:
Lowercase on my request for better readability (and to save some horizontal space)

Reply:
With seconds in the timestamp the footer format is very cramped, especially as the number of pages goes over 100. The full timestamp (up to the nanosecond if we want) is always in the EML anyway.